bundles/http/Citrix/Citrix-Podio/metadata.json

```
@@ -1,36 +1,36 @@
 {
   "type" : "HTTP",
   "vendor" : "Citrix",
-  "id" : "00000017-0000-0000-0000-000000000000",
+  "id" : "25c31592-1910-4603-a21a-fbd2f61894bf",
   "trackingUuid" : "76ee7428-3669-4595-8e8f-dada937ab765",
   "title" : "Citrix Podio",
   "description" : "Integrate with Podio to deliver quick actions on Citrix Workspace utilizing the flexibility and diversified use cases on Podio. With this integration, you can easily connect our out-of-the-box microapps on Workspace to the corresponding Podio apps readily available on the Podio App Market, or your customized apps on Podio.",
-  "iconUrl" : "https://iws-prod-global-customicons-cdn-endpoint.azureedge.net/image/pjBX0919Ni9PyI15Wx2qvX1TAt10pvpt.png",
-  "masVersion" : "1.77.0",
+  "iconUrl" : "https://microappsicontest-cdn.azureedge.net/image/M8T9gFzPCyxSiOdEAZ0Eq2Xesr4LQ2it.png",
```

This is a test URL. Should be changes for the old one. Also needs to be changed in the `file.sapp`.
```suggestion
  "iconUrl" : "https://iws-prod-global-customicons-cdn-endpoint.azureedge.net/image/pjBX0919Ni9PyI15Wx2qvX1TAt10pvpt.png",
```